apps/root/webpack.dev.js

```diff
@@ -1,15 +1,25 @@
 const { merge } = require('webpack-merge');
 const common = require('./webpack.common.js');
+const webpack = require('webpack');
 
-module.exports = merge(common, {
-  mode: 'development',
-  devtool: 'inline-source-map',
-  devServer: {
-    port: 3000,
-    historyApiFallback: true,
-    hot: true,
-    client: {
-      overlay: false,
+const SpeedMeasurePlugin = require('speed-measure-webpack-plugin');
+const smp = new SpeedMeasurePlugin();
+
+module.exports =
+  // smp.wrap({...
+  merge(common, {
+    mode: 'development',
+    devtool: 'inline-source-map',
+    devServer: {
+      port: 3000,
+      historyApiFallback: true,
+      hot: true,
+      client: {
+        overlay: false,
+      },
+    },
+    snapshot: {
+      managedPaths: [/^(.+?[\\/]node_modules[\\/](?!(ui-library|common-types))(@.+?[\\/])?.+?)[\\/]/],
```

Comment:
what's this for? So it can pick up changes on the other repos?

Reply:
Exactly, I followed [this documentation](https://webpack.js.org/configuration/other-options/#managedpaths)
Hot reload stopped working as it did before; I think this might be because it now builds the frontend before `build:packages` finishes. This configuration suits our use case

Reply:
Ok so having a watch on those packages still trigger a re-render right?

Reply:
Yes, the server restarts on any changes from this directory. Running build:packages triggers the server a couple of times, due to the deletion and creation of distribution files.
I have updated it in the new commit, using a more modern property (`unmanagedPaths`). This managedPaths output something extra in the console, this is cleaner and more maintainable for any new package